controllers/machine_controller_phases.go

```diff
@@ -192,6 +200,9 @@ func (r *MachineReconciler) reconcileBootstrap(ctx context.Context, cluster *clu
 	if err != nil {
 		return ctrl.Result{}, err
 	}
+	if externalResult.RequeueAfter > 0 {
+		return ctrl.Result{RequeueAfter: externalResult.RequeueAfter}, nil
+	}
```

Comment:
The use of `external.ReconcileOutput` here makes this quite confusing. Not sure I have any good suggestions on how to make it less confusing without requiring more invasive refactoring, though :(